src/handlers/prioritize.rs

```diff
@@ -39,18 +39,23 @@ impl Handler for PrioritizeHandler {
                         // We need to take the exact same action in this case.
                         return Ok(Some(Prioritize::Start));
                     } else {
-                        match glob::Pattern::new(&config.priority_labels) {
-                            Ok(glob) => {
-                                let issue_labels = event.issue().unwrap().labels();
-                                let label_name = &e.label.as_ref().expect("label").name;
+                        let label_name = &e.label.as_ref().expect("label").name;
```

Comment:
Maybe extract the label name to avoid `.as_ref().expect()` twice?

Reply:
:+1:

Reply:
Done.